lib/animina_web/potential-partner.ex

```diff
@@ -0,0 +1,20 @@
+defmodule AniminaWeb.PotentialPartner do
```

Comment:
Shouldn't be the filename `potential_partner.ex`and not `potential-partner.ex`? I am not a 100% sure but `_` feels better to me.